src/content/dnd/posts/2023-05-10.md

```diff
@@ -0,0 +1,175 @@
+- The two walk back together and go into the inn. Brooke locks the door behind them.
+- [Oneshoe](/dnd/characters/oneshoe/) has been tailing them unnoticed. She sneaks up to a window, quietly opens its, and slips inside.
+- In Finn's head, [Magrin](/dnd/npcs/magrin/) expresses some frustration that he hasn't had a chance to "fuck up some devils" lately.
+- Brooke pours each of them a very large glass of wine. It seems quite strong, but Finn downs it. They have another.
+- Oneshoe notices that Brooke may have slipped something into Finn's drink. Still, she chooses to wait and observe further.
+- Brooke starts asking some questions and seems to show a great interest in learning more about about [Gray](/dnd/characters/haeltin-var-astora/), whom Brooke pointedly refers to as "Hael".
+    - (Finn previously let Gray's real name slip to Brooke back in the spa episode.)
+- Finn doesn't seem to react to the usage of his true name. However, instead of answering Brooke, Finn turns the conversation to Bailey and Not Bailey. She starts enumerating her friends.
+- Finn asks if Brooke knows [Gimble](/dnd/characters/gimble-the-diviner/). Brooke claims she doesn't know anyone by that name.
+- Oneshoe decides to snoop about a bit and begins to pad softly along the ceiling with her *Slippers of Spider Climbing*. Unfortunately, she brushes against a burr in the wood and makes a slight noise.
+- Brooke takes notice and calls out saying, "You should come on down and join us for a drink."
+- Oneshoe gracefully slips down and snatches the glass away from Finn.
+- They exchange a few words, but it is not long before Oneshoe bristles at Brooke.
+- Oneshoe throws Finn over her shoulder and exits out the window like a boss.
+
+#### Gray Surveys the Basement
+
+- Though it has been a scant few minutes since an undead horde had attacked the manor, Gray and [Horns](/dnd/characters/horns/) are surprised to find themselves alone.
+- Gray goes down to the basement to check if the cistern is still fouled. He finds that the waters have cleared, and in the bottom of the pool sits a *Decanter of Endless Water*, which he retrieves. Strangely, no trace of the rusalka remains.
+- Perhaps wondering what was real, Gray investigates the area and finds a very real and very dead raven. He employs *Grim Psychometry* upon the form of the crow to see what he can learn, this time expending some of his own blood to augment the rite. After a moment, the watercolor tattoos upon his arms begin to animate and project light upon the wall. It seems The Nameless Wolf is granting a vision. Gray sees the porcelain mask of the Raven Queen and hears the flutter of wings. Then there is another familiar face, one that Gray has not seen in a very long time. He sees the face of his mother, but just for the briefest moment.
+- At seeing the flash of his mother's face, Gray has a strong feeling that he should not trust the white (?) raven.
```

Comment:
Is this correct? Or was it the other raven that should not be trusted?

Reply:
The white-faced raven is the one that gave Gray the warning.